webapp/src/app.tsx

```diff
@@ -4096,7 +4096,27 @@ function initPacketIO() {
                 id: 'n/a', // TODO
                 data
             }, "*")
+        },
+        (type, payload) => {
+            if (type == pxt.HF2.CUSTOM_EV_JACDAC) {
+                window.postMessage({
+                    type: "messagepacket",
+                    broadcast: false,
+                    channel: type,
+                    data: payload,
+                    sender: "packetio",
+                }, "*")
+            }
         });
+
+    window.addEventListener('message', (ev: MessageEvent) => {
+        const msg = ev.data
+        if (msg.type == 'messagepacket' && msg.channel == pxt.HF2.CUSTOM_EV_JACDAC && msg.sender != "packetio")
```

Comment:
same thing here, look up channel in ``pxt.appTarget.simulator.messageSimulators``